cartography/intel/aws/ec2/instances.py

```diff
@@ -245,6 +244,39 @@ def load_ec2_instances(
                     ).consume()  # TODO see issue 170
 
             load_ec2_instance_network_interfaces(neo4j_session, instance, update_tag)
+            load_ec2_instance_ebs_volumes(neo4j_session, instance, update_tag)
+
+
+@timeit
+def load_ec2_instance_ebs_volumes(neo4j_session: neo4j.Session, instance: Dict, update_tag: int) -> None:
+    instanceId = instance["InstanceId"]
+    if 'BlockDeviceMappings' in instance and len(instance['BlockDeviceMappings']) > 0:
+        ingest_volume = """
+        MERGE (v:EBSVolume{id: {VolumeId})
+        ON CREATE SET v.firstseen = timestamp()
+        SET v.lastupdated = {update_tag}, v.deleteontermination = {DeleteOnTermination}
+        WITH v
+        MATCH (aa:AWSAccount{id: {AWS_ACCOUNT_ID}})
+        MERGE (aa)-[r:RESOURCE]->(v)
+        ON CREATE SET r.firstseen = timestamp()
+        SET r.lastupdated = {update_tag}
+        WITH v
+        MATCH (instance:EC2Instance{instanceid: {InstanceId}})
+        MERGE (v)-[r:ATTACHED_TO]->(instance)
+        ON CREATE SET r.firstseen = timestamp()
+        SET r.lastupdated = {update_tag}
+        """
+
+        ebs_mappings = instance['BlockDeviceMappings']
+        for mapping in ebs_mappings:
+            if 'VolumeId' in mapping['Ebs']:
+                neo4j_session.run(
```

Comment:
Instead of calling the neo4j_session.run inside a loop, can we use unwind based approach here?

Reply:
Check my comment on a similar issue for the snapshot bit below.

Reply:
We should follow same approach as the snapshots here as well

Reply:
Okay!